src/views/Game/GameLogModal.tsx

```diff
@@ -38,15 +39,17 @@ interface LogEntry {
 }
 
 export class GameLogModal extends Modal<Events, GameLogModalProperties, { log: Array<LogEntry> }> {
+    config: any;
+
     constructor(props) {
         super(props);
 
         this.state = {
             log: [],
         };
 
-        const config = this.props.config;
-        const game_id = config.game_id;
+        this.config = this.props.config;
```

Comment:
I don't think it will matter for a modal, but saving props to a class variable can cause issues.  The reason being that if `config` changes in the parent component, the changes will not propagate to `GameLogModal` since we have saved the old config as a class member and continue to reference that.

I've built a small code sandbox to try to illustrate how changes fail to propagate if you only reference `this.config` but don't update it when props changes: https://codesandbox.io/s/react-playground-forked-i7ofrh?file=/index.js

Reply:
Right I guess that makes sense. I will revert to the `const` version. I didn’t intentionally change it in any case, but I also didn’t realise the example you’re pointing out. It’s good to know.

Reply:
updated now, let me know if you spot something else.